content/en/containers/kubernetes/control_plane.md

```diff
@@ -423,16 +453,17 @@ annotations:
 
 ## Kubernetes on OpenShift 4 {#OpenShift4}
 
-### Using the Operator (v1.18.0+)
 <div class="alert alert-info">This feature is in Preview.</div>
 
-The Datadog Operator can automatically configure monitoring for Kubernetes control plane components including the API Server, etcd, Controller Manager, and Scheduler.
+Datadog supports monitoring Kubernetes Control Plane components, including the API Server, etcd, Controller Manager, and Scheduler.
 
-#### Prerequisites
+{{< tabs >}}
+{{% tab "Datadog Operator" %}}
```

Comment:
Just noting there are a couple of broken link on this tab:

1. `[datadog-operator Helm chart][12]`. 
2. `Or, for OpenShift users who installed the operator through OperatorHub/Marketplace (the recommended method)`

For links that are on a tab, the link reference must be contained within the tab. For example, for `[datadog-operator Helm chart][12]`, you would put `[12]: https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator` before the closing `{{% /tab %}}` element.